### ISSUE TITLE
Scalable approach to incorporating ramp rate constraints

### ISSUE THREAD

**Issue by moriartyjm** [moriartyjm/netzerotc#4]
At workshop 2, a number of possible ramp rate constraints were discussed - eg. on construction rates, and also rate of change in jobs (particularly job decreases). 

A scalable, or at least clear, approach to imposing ramp rates would therefore be helpful.

**Comment by blairoconnornztc** [comment]
Hi John, I think a max of 25,000 job losses in a year would be a limit. The industry lost roughly that number during the pandemic which had a significant impact and led to restructuring of a number of businesses. I also think there should be a constraint the following year if the number of job losses reaches near/or close to the limit as continued job losses on a year to year cycle would cause major issues. e.g. can only be 50% (12.5) if the 25,000 limit is reached.  I'd be interested to hear your thoughts on creating a constraint for the number of years in which a deficit for jobs can be utilised e.g. 2 years of job losses. 


I'm keen to hear Tom and Gavin's views.

**Comment by moriartyjm** [comment]
Yes it's possible to include multiple constraints on the same quantity - so eg. a solution is feasible only if both condition 1 and condition 2 are met. Here condition 1 could be per year, condition 2 could be across multiple years

**Comment by blairoconnornztc** [comment]
That would work. I do think ensuring the volatility of change needs to be included and without having both conditions 1 and 2 would represent an issue in the real world.
